requirements-test.txt

```diff
@@ -7,3 +7,4 @@ boto3
 jinja2
 PyGithub
 mbed-tools-ci-scripts
+mbed-host-tests
```

Comment:
requirements-test.txt is for testing or testing and run-time both? As currently implemented, it looks like mbed-host-tests is a normal, non-testing, requirement. Is there a requirements.txt yet?

Reply:
Package runtime dependencies are specified in `setup.py`. This is just for testing.

Reply:
`mbed-tools` doesn't depend on `mbed-host-tests` anymore.